### PR TITLE
new translations for upcoming release

### LINKED ISSUE #4764: translation checks error on bash
Hi @MichaelChirico,
I'm running CRAN_Release.cmd and I get the following error : 
```
~/GitHub/data.table$ for MSG in error warning DTWARN DTPRINT Rprintf STOP Error;
>   do for SRC_FILE in src/*.c;
>     # no inplace -i in default mac sed
>     do sed -E "s/$MSG[(]("[^"]*")/$MSG(_(\1)/g" $SRC_FILE > out;
bash: syntax error near unexpected token `)'
~/GitHub/data.table$
```
Any ideas?  Probably some difference between Mac and Linux bash?
Thanks.

### PR DESCRIPTION
**Pull request by MichaelChirico**
Closes #4764

### CODE REVIEW

**Comment by MichaelChirico** [comment]
About 20 translations we rely on our team for, have sent out an ask.

File: src/forder.c [hunk 1]
```diff
@@ -501,7 +501,7 @@ SEXP forder(SEXP DT, SEXP by, SEXP retGrpArg, SEXP sortGroupsArg, SEXP ascArg, S
   int keyAlloc = (ncol+n_cplx)*8 + 1;         // +1 for NULL to mark end; calloc to initialize with NULLs
   key = calloc(keyAlloc, sizeof(uint8_t *));  // needs to be before loop because part II relies on part I, column-by-column.
   if (!key)
-    STOP("Unable to allocate %"PRId64" bytes of working memory", (uint64_t)keyAlloc*sizeof(uint8_t *));  // # nocov
+    STOP(_("Unable to allocate %"PRIu64" bytes of working memory"), (uint64_t)keyAlloc*sizeof(uint8_t *));  // # nocov
```
Comment (MichaelChirico) [inline src/forder.c:504]: @mattdowle this fix is correct right?

File: po/zh_CN.po [hunk 2]
```diff
@@ -3386,19 +3389,21 @@ msgstr ""
...
 "read ?fwrite for nThread, buffMB and verbose options."
-msgstr ""
+msgstr "无法分配 %d MB * %d 的线程压缩缓存；'%d: %s'。请"
+"阅读 ?fwrite 中对 nThread、buffMB 和 verbose 选项的说明。"
 
 #: fwrite.c:851 fwrite.c:883 fwrite.c:885
 #, c-format
 msgid "z_stream for data (%d): "
-msgstr ""
+msgstr "data （%d）的 z_stream："
```
Comment (jangorecki) [inline po/zh_CN.po:3406]: Double whitespace here :)

Comment (MichaelChirico) [reply po/zh_CN.po:3406]: it's actually just one :) but the `(` is a wide one from the chinese code range: `（`

File: src/forder.c (hunk 1 above)
Comment (mattdowle) [reply src/forder.c:504]: @MichaelChirico oh yes, good spot!

**Comment by jangorecki** [comment]
If it is ready to merge we could have it now for a hot fix needed due to #4769, adding milestone

**Comment by MichaelChirico** [comment]
yes it's ready

**Pull request merged**